open-metadata-implementation/user-interfaces/presentation-server/cra-server/router/routes.js

```diff
@@ -0,0 +1,275 @@
+    instance
+      .post("", body)
+      .then(function (response) {
+        console.log("response.data");
+        console.log(response.data);
+        const resBody = response.data;
+        res.setHeader("Content-Type", "application/json");
+        res.json(resBody);
+      })
+      .catch(function (error) {
+        console.log(error);
+        res.status(400).send(error);
+      })
+      .then(function () {
+        // always executed
+      });
+  } else {
+    res.status(400).send("Error, invalid supplied URL: " + incomingUrl);
+  }
+});
+
+/**
+ * Middleware to proxy put requests that start with /servers.
+ * The outbound call is made with https. 
+ */
+router.put("/servers/*", (req, res) => {
+  const incomingUrl = req.url;
+  console.log("/servers/* put called " + incomingUrl);
+  const body = req.body;
+  console.log("Got body:", body);
+  const servers = (req.app.get('servers'));
+  if (validateURL(incomingUrl, servers)) {
+    const instance = getAxiosInstance(incomingUrl);
+    instance
+      .put("", body)
+      .then(function (response) {
+        console.log("response.data");
+        console.log(response.data);
+        const resBody = response.data;
+        res.setHeader("Content-Type", "application/json");
+        res.json(resBody);
+      })
+      .catch(function (error) {
+        console.log(error);
+        res.status(400).send(error);
+      })
+      .then(function () {
+        // always executed
+      });
+  } else {
+    res.status(400).send("Error, invalid supplied URL: " + incomingUrl);
+  }
+});
+
+/**
+ * Middleware to proxy delete requests that start with /servers.
+ * The outbound call is made with https. 
+ */
+router.delete("/servers/*", (req, res) => {
+  const incomingUrl = req.url;
+  console.log("/servers/* delete called " + incomingUrl);
+  const servers = (req.app.get('servers'));
+  if (validateURL(incomingUrl, servers)) {
+    const instance = getAxiosInstance(incomingUrl);
+    instance
+      .delete()
+      .then(function (response) {
+        console.log("response.data");
+        console.log(response.data);
+        const resBody = response.data;
+        res.setHeader("Content-Type", "application/json");
+        res.json(resBody);
+      })
+      .catch(function (error) {
+        console.log(error);
+        res.status(400).send(error);
+      })
+      .then(function () {
+        // always executed
+      });
+  } else {
+    res.status(400).send("Error, invalid supplied URL: " + incomingUrl);
+  }
+});
+
+/**
+ * Middleware to proxy get requests that start with /servers.
+ * The outbound call is made with https. 
+ */
+router.get("/servers/*", (req, res) => {
+  const url = req.url;
+  console.log("/servers/* get called " + url);
+  const servers = (req.app.get('servers'));
+  if (validateURL(url, servers)) {
+    const instance = getAxiosInstance(url);
+    instance
+      .get()
+      .then(function (response) {
+        console.log("response");
```

Comment:
we may need to guard this or remove it if any potential for personally identifiable information

Reply:
is anything in the response sensitive in any way? Who sees this log? is it just the user accessing the ui? or the server admin? The former is ok, the latter requires care.

Reply:
@davidradl ?

Reply:
Commented out lines in question